src/components/explore/exploreList/ExploreList.tsx

```diff
@@ -40,8 +40,12 @@ const ItemRightElement = ({
 
   return (
     <Typography color="inverted-text" fontSize="small">
-      {t('explore:by', {
-        provider: provider,
+      {provider &&
+        `
```

Comment:
I would be tempted not to make a return here.

Reply:
Yep, prettier killed me 😜